CSS tweaks for icons in the Navigation block

mattrice opened the pull request:
A small tweak to the CSS in the Navigation block.
Before:
![pull_request_before](https://f.cloud.github.com/assets/3780536/2208662/6c04fad4-9982-11e3-9c0c-fbc671659222.PNG)
After:
![pull_request_after](https://f.cloud.github.com/assets/3780536/2208665/706ce5a0-9982-11e3-8a6d-7608e2c10eb6.PNG)

mattrice committed:
CSS tweaks for icons in the Navigation block
-My profile/People in Participation folder use the fa-user icon
-Show open/closed folders when branches expanded/collapsed

vaughany commented:
:+1:

mattrice committed:
Added required (Moodle) version point number

mattrice committed:
Rollup of origin source changes

gjb2048 committed:
V2.7.0.1

gjb2048 committed:
V2.7.0.1

DBezemer committed:
Update translations and start dutch translation

DBezemer committed:
Commit various fixes and typos

DBezemer committed:
Code style and whitespace cleanup

gjb2048 committed:
Merge branch 'master' of github.com:DBezemer/moodle-theme_essential

Conflicts:
	style/font-awesome.css

Commit:
Update gitignore

Commit:
Bump version number and sanitize requirements

DBezemer committed:
Align version.php with Julians scheme slightly

DBezemer committed:
Update Readme and version requirement

DBezemer committed:
Language cleanup, and WIP slideshow improvements

DBezemer committed:
Add Dutch Translation and cleanup English

DBezemer committed:
Remove default background image, place header bar at top, re-align slideshow, add new analytics

DBezemer committed:
Updating Slideshow design to offer a more fluent experience.

DBezemer committed:
Attempt to cleanup admin menu (failed due to MDL-46259)

DBezemer committed:
Bump version to Moodle 2.7 and update README

gjb2048 committed:
Merge branch 'MOODLE_26' of github.com:DBezemer/moodle-theme_essential

Conflicts:
	version.php

gjb2048 committed:
Put back Pirate!

Commit:
Fix #4, place background in correct position

DBezemer committed:
Fix #6, further slideshow improvements, release 2.7.2

DBezemer committed:
Bump version number for release 2.7.2

DBezemer committed:
Small change in image stretching size.

gjb2048 committed:
Beginning of LESS conversion and update to FontAwesome 4.1.0.

DBezemer committed:
Fix #8

DBezemer committed:
Merge pull request #10 from gjb2048/master_less_grunt

Beginning of LESS conversion and update to FontAwesome 4.1.0.

DBezemer committed:
Merge small missing changes for less conversion

DBezemer committed:
re-add some minor css fixes and remove nojs from default less compilation (only loaded in noscript tags in frontpage.php).

Commit:
Update missing colours from colourswitcher, resolve padding on mobile views

Commit:
Bump version number for master branch

gjb2048 committed:
Merge branch 'master' of github.com:DBezemer/moodle-theme_essential

Conflicts:
	style/essential.css

DBezemer committed:
Make sure base is a fully empty layout, fix #16

DBezemer committed:
Large cleanup #1, not otherwise specified

gjb2048 committed:
Merge branch 'master' of github.com:DBezemer/moodle-theme_essential

gjb2048 committed:
Carousel replacement for slider.  Issue #18.

gjb2048 committed:
Merge pull request #21 from DBezemer/master_carousel

Master carousel

gjb2048 committed:
Update README.txt for issue #18.

gjb2048 commented:
Looks good, would you be able to rebase, squash and resubmit here please: https://github.com/DBezemer/moodle-theme_essential/issues

DBezemer committed:
Massive rework and cleanup, see Readme, thanks to @gjb2048 for the awesome new slider!

DBezemer committed:
Fix #20

gjb2048 committed:
Slight fixes for the carousel.

DBezemer committed:
Merge pull request #27 from DBezemer/master_carousel_fix

Slight fixes for the carousel.

DBezemer committed:
License info cleanup, some minor CSS fixes, new dock mod, all tabs to spaces.

DBezemer committed:
Only use dock.js for Moodle 2.7 and up

Commit:
Revert back to theme specific font loading for IE8 compatibility, see: MDL-46358
Also adding back CDN loading, and separating fontawesome and essential css.

Commit:
Remove map file, developers can generate this running grunt tasks.

Commit:
Resolve duplicate id, fix #30 and #31

Commit:
Also push CSS for previous fixes

gjb2048 committed:
Docking fixes and efficiency for slider.

gjb2048 committed:
Alert on frontpage only.

gjb2048 committed:
Alternative theme colours for slider.

DBezemer committed:
See README.txt, main changes: breadcrumb and fixes

DBezemer committed:
Fix #40, add a floating save button on admin save pages.

DBezemer committed:
Resolve #40 in a more reliable way.

Commit:
Fix header bug (spans did not match all possible cases). Also make sure the lib cases are optimized by breaking out when possible.

Commit:
Prevent creation of empty menu link

Commit:
Correct indentation and close all div tags.

Commit:
Set correct header height when no logo is present to prevent difference between login and no-login.

Commit:
Fix check for breadcrumb style

Commit:
Align logo so it fits better in the header

Commit:
Whoops @ quotes

Commit:
Align logo

Commit:
Stick classic navbar to top on scroll as well, resolve font check if Google Fonts are not used. Rework plugin structure.

Commit:
It is important not to forget the style!

Commit:
Add popup file, hopefully fix #44, will test more later

Commit:
Correctly disable custom icons

gjb2048 committed:
Fix for #47.

gjb2048 committed:
Merge branch 'master' of github.com:DBezemer/moodle-theme_essential

DBezemer committed:
Fix for #46, looks aligned for me, but please test on Moodle 2.6

DBezemer committed:
Fix #44

DBezemer committed:
Fix style for login block and small optimize for secure layout.

DBezemer committed:
Update loading icons and remove margin on header when viewing on mobile.

DBezemer committed:
Bump version number for release

DBezemer committed:
Update README for release

DBezemer committed:
See #49

DBezemer committed:
Remove language check for alerts, preventing warnings/errors

gjb2048 committed:
Add text colour.

Conflicts:
	style/essential.css

gjb2048 committed:
Fix icon colour in line with SVG's.

Conflicts:
	style/essential.css

gjb2048 committed:
Don't compress as Moodle does in the cache and makes it harder to debug otherwise.

Conflicts:
	style/essential.css

gjb2048 committed:
Fix for #68

gjb2048 committed:
Slider fixes.

gjb2048 committed:
Merge pull request #75 from gjb2048/master_#68

Fix for #68

gjb2048 committed:
Recompile CSS after rebase.

gjb2048 committed:
Merge pull request #62 from gjb2048/master_text_colour

Add text colour.

gjb2048 committed:
Add in svgmin and revert icons to gray.

gjb2048 committed:
Fix colours work in progress.

DBezemer committed:
Update theme_essential.php

Fix #79 as Mary honors her username

gjb2048 committed:
Fix alternative colour sets.

gjb2048 committed:
Merge branch 'master' of github.com:DBezemer/moodle-theme_essential

gjb2048 committed:
Moodle 2.6 does not support IE8 and remove duplicate page settings require.

gjb2048 committed:
Carousel slider caption below image option.

gjb2048 committed:
Bump to V2.7.3.1 - BETA test version.

gjb2048 committed:
V2.7.4a - Pre-release for testing.

gjb2048 committed:
2.7.4a

gjb2048 committed:
Fix for #78.  Migrated manual RTL CSS to own sheet.  Fixed header RTL logic.

gjb2048 committed:
Add rtl.css.

gjb2048 committed:
Fix #85.

gjb2048 committed:
Remove redundant plugin css exclusion that no longer exists.  Thanks to Mary Evans.

gjb2048 committed:
Fix #88 and #90.

gjb2048 committed:
Fix issue #89 and navigation bar with custom menu items on a narrow width page setting.

gjb2048 committed:
2.7.4d - Issue #93.

gjb2048 committed:
Improve slider description.

gjb2048 committed:
2.7.4

gjb2048 committed:
Merge branch 'master' of github.com:gjb2048/moodle-theme_essential

sk-unikent committed:
Lang en/iosicondesc - Change 'them' to 'theme'

gjb2048 committed:
Merge pull request #98 from sk-unikent/patch-1

Lang en/iosicondesc - Change 'them' to 'theme'

gjb2048 committed:
Start of 2.7.5.

gjb2048 committed:
Fix #107. No fixed width to header on login page.

gjb2048 committed:
Fix issue #101: Navbar not expanding properly on android mobile and tablet.

gjb2048 committed:
Style fix for #107.

gjb2048 committed:
Fix issue #96.  2.7.4 breaks oldnavbar setting.

gjb2048 committed:
Update 2.7.5 readme.

gjb2048 committed:
Fix 'i' tag with no logo.

DBezemer committed:
- FIX: Issue #113: Remove FontAwesome from all links in section heading except for the edit icon
- FIX: Issue #110: Refactor middle blocks wording, thanks @mkpelletier
- FIX: Issue #109: Re-order middle blocks settings, thanks @mkpelletier
- FIX: Issue #108: Add similar tggle options to front page content, thanks @mkpelletier
- FIX: Issue #67:  Refactored so menus are renderer all separately
- FIX: Issue #60:  Add styling to invisible courses in breadcrumb
- FIX: Correctly show no enrolments message when all courses are hidden
- FIX: Reworked header menu and added responsive options to apps/social icons
- FIX: Removed dnd upload status as it blocks the edit icons
- FIX: Further work on RTL support
- NEW: Removed bootstrapbase dependency for further minification
- NEW: Split out alternative theme colors to reduce CSS loading when this is not enabled
- NEW: Small tweak to site widths, now available options are 1400px, 1200px and 960px
- NEW: Switched to bootstrap all in one for massive speed improvement (from 10 to 1 http request)
- NEW: Removed option for restricting block width as it was not possible tog et this working properly

gjb2048 committed:
Issue #114.

Commit:
Fix #92 and give main content a white bg

Commit:
Merge branch 'master' of https://github.com/DBezemer/moodle-theme_essential

Commit:
Use optimized dot png and fully fix #92

Commit:
Remove IE8 message as it works fine in IE8

DBezemer committed:
Sort out some less files to optimize for small releases

DBezemer committed:
Correctly reflect merging of font-awesome css

DBezemer committed:
Compress flipped css

DBezemer committed:
Minor changes (jk)

DBezemer committed:
Small fixes cleanup

DBezemer committed:
Responsive changes to dock and cleanup of header.

DBezemer committed:
fixing small issue when user has no courses assigned yet.
Cleanup of footer as pagesettings is always included already

DBezemer committed:
Fix #81 and merge RTL changes

DBezemer committed:
And stay gone!

gjb2048 committed:
Fix a setting in categories.less and continuance of #1.

gjb2048 committed:
Issue #123.

gjb2048 committed:
Fix #126.

gjb2048 committed:
Fix: https://moodle.org/mod/forum/discuss.php?d=264952#p1151378

gjb2048 committed:
Fix message dropdown menu stays open when '.open' applied to dropdown.

gjb2048 committed:
Fix #127.

gjb2048 committed:
Progress on dropdown colours.  Alternatives not finished yet.

DBezemer committed:
Fix #131 #130 #129 and cleanup renderer, language and settings

DBezemer committed:
Day of the boo-boo

DBezemer committed:
And again!

DBezemer committed:
Third times a charm

DBezemer committed:
Enhance breadcrumb settings, cleanup settings and correctly fix category icons.

DBezemer committed:
Update lib.php with changes and align code for performance output

Commit:
Language support for Messages timing

Commit:
Add capability support to the grades item.
TODO: add documentation to readme to enable grade overview capability on site level.

Commit:
provide My Grades regardless of Moodle's crappy implementation

Commit:
further optimize context loading

Commit:
Indentation fixes

Commit:
Align naming of course renderer file.

Commit:
Add thirdpartylibs.xml and fix #42

Commit:
Fix #5 and include mixing in Essentials for further future optimization (Essential bootswatch maybe?)

DBezemer committed:
Fix #52

gjb2048 committed:
Improve navbar LESS and optimise, continuation of #1.

gjb2048 committed:
Slight tweak to messages code for screen readers.

DBezemer committed:
Add donation link to intro/header/install info

DBezemer committed:
Uniform borders between course and blocks

gjb2048 committed:
Fix span in span without row-fluid.

gjb2048 committed:
Fix navbar colours.

gjb2048 committed:
Fix #134.

gjb2048 committed:
Fix #135.

gjb2048 committed:
Fix 136.

mattrice committed:
Merge branch 'master' of https://github.com/DBezemer/moodle-theme_essential
